CefSharp.Wpf/Rendering/InteropBitmapInfo.cs

```diff
@@ -31,14 +32,24 @@ public override void ClearBitmap()
 
         public override void Invalidate()
         {
-            Bitmap.Invalidate();
+            if (Bitmap != null)
+            {
+                Bitmap.Invalidate();
+            }
         }
 
         public override BitmapSource CreateBitmap()
         {
             var stride = Width * BytesPerPixel;
 
-            Bitmap = (InteropBitmap)Imaging.CreateBitmapSourceFromMemorySection(FileMappingHandle, Width, Height, PixelFormat, stride, 0);
+            if (FileMappingHandle == IntPtr.Zero)
+            {
+                ClearBitmap();
```

Comment:
Close :smile: I'll remove the else (redundant) and move the stride calculation after the `if` when it's eventually time to merge this :+1: